### ISSUE TITLE
missing "context"

### ISSUE THREAD

**Issue by andrzejsliwa** [jarednorman/plugsnag#11]
Hi, I think here is missing most important part of error report which is conn, the state 

``` elixir
      defp handle_errors(_conn, %{reason: exception}) do
        Bugsnag.report(exception)
      end
```

we can make it as: 

``` elixir
        metadata = %{"host" => conn.host,
                     "method" => conn.method,
                     "path_info" => conn.path_info,
                     "request_path" => conn.request_path,
                     "query_string" => conn.query_string,
                     "port" => conn.port,
                     "scheme" =>  conn.scheme,
                     "peer" => inspect(conn.peer),
                     "remote_ip" => inspect(conn.remote_ip),
                     "req_headers" => inspect(conn.req_headers)}
        Bugsnag.report(exception, metadata: metadata)
```

**Comment by mootpointer** [comment]
Seems like a good idea. I've been thinking about a way of allowing extra information to be sent, but this seems like a reasonable first step.

Would you be willing to submit a PR, or shall I have a hack on it when I have time?

**Comment by mootpointer** [comment]
Looking at it a bit further... I think metadata should be something like:

``` elixir
metadata = %{
  "request_info" => %{
    "host" => conn.host,
    "method" => conn.method,
    "path_info" => conn.path_info,
    "request_path" => conn.request_path,
    "query_string" => conn.query_string,
    "port" => conn.port,
    "scheme" =>  conn.scheme,
    "peer" => inspect(conn.peer),
    "remote_ip" => inspect(conn.remote_ip),
    "req_headers" => inspect(conn.req_headers)
   }
}

```

This will put it all on the same tab.

I think it's probably worthwhile formatting `peer`, `remote_ip` and `req_headers` nicely.

**Comment by mootpointer** [comment]
Have a look at #12 and let me know what you think.

**Comment by ku1ik** [comment]
If I understand correctly this was already implemented in #12 and released. We can probably close this?
